plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -2059,6 +2059,10 @@ producer_plugin_impl::determine_pending_block_mode(const fc::time_point& now,
    // If the next block production opportunity is in the present or future, we're synced.
    if (!_production_enabled) {
       _pending_block_mode = pending_block_mode::speculating;
+      if (_producers.find(scheduled_producer.producer_name) != _producers.end()) {
+         fc_elog(_log, "Not producing block because stale production not enabled, block ${t}", ("t", block_time));
```

Comment:
Do you want to log the producer name?

Reply:
I thought about it, but none of the other `Not producing block because` log statements log the producer. I didn't want to change any of the existing ones and just kept this consistent. Under normal operation the BP should know their one and only one producer name.